### PR TITLE
[popsift] fix copy tools

### PR DESCRIPTION
**Pull request by simogasp**
- #### What does your PR fix?  
Popsift was not installing the dlls in bin folder because the bin folder was just moved into the tools directory.
Properly using `vcpkg_copy_tools()` instead of managing the file manually fixes the problem.
Incidentally, it adds the now required `FEATURES` to `vcpkg_check_feature()`

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

File: ports/popsift/portfile.cmake [hunk 1]
```diff
@@ -29,13 +30,10 @@ vcpkg_copy_pdbs()
...
- # move the bin direcory to tools
+ # copy the apps in tools directory
  if ("apps" IN_LIST FEATURES)
-    file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/tools)
-    file(RENAME "${CURRENT_PACKAGES_DIR}/bin" ${CURRENT_PACKAGES_DIR}/tools/popsift)
-    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/bin")
-#    file(RENAME "${CURRENT_PACKAGES_DIR}/debug/bin" ${CURRENT_PACKAGES_DIR}/tools/popsift/debug)
-    vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/popsift)
+     set(POPSIFT_TOOLS popsift-demo)
+     vcpkg_copy_tools(TOOL_NAMES ${POPSIFT_TOOLS} AUTO_CLEAN)
```
Comment (ras0219) [inline ports/popsift/portfile.cmake:36]:
```suggestion
     vcpkg_copy_tools(TOOL_NAMES popsift-demo AUTO_CLEAN)
```

**Review by ras0219 — APPROVED** [review]

File: ports/popsift/portfile.cmake [hunk 2]
```diff
@@ -12,6 +12,7 @@ vcpkg_find_cuda(OUT_CUDA_TOOLKIT_ROOT CUDA_TOOLKIT_ROOT)
 message(STATUS "CUDA_TOOLKIT_ROOT ${CUDA_TOOLKIT_ROOT}")
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
     apps       PopSift_BUILD_EXAMPLES
```
Comment (NancyLi1013) [inline ports/popsift/portfile.cmake:16]:
```suggestion
    FEATURES
        apps       PopSift_BUILD_EXAMPLES
```

File: ports/popsift/portfile.cmake [hunk 3]
```diff
@@ -12,6 +12,7 @@ vcpkg_find_cuda(OUT_CUDA_TOOLKIT_ROOT CUDA_TOOLKIT_ROOT)
...
 )
```
Comment (NancyLi1013) [inline ports/popsift/portfile.cmake:18]: Could you please update the format

`OPTIONS ${FEATURE_OPTIONS} -DCUDA_TOOLKIT_ROOT_DIR=${CUDA_TOOLKIT_ROOT}`
as
```
OPTIONS ${FEATURE_OPTIONS} 
    -DCUDA_TOOLKIT_ROOT_DIR=${CUDA_TOOLKIT_ROOT}
```
?

File: ports/popsift/portfile.cmake [hunk 4]
```diff
@@ -29,13 +30,10 @@ vcpkg_copy_pdbs()
...
-    file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/tools)
-    file(RENAME "${CURRENT_PACKAGES_DIR}/bin" ${CURRENT_PACKAGES_DIR}/tools/popsift)
-    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/bin")
-#    file(RENAME "${CURRENT_PACKAGES_DIR}/debug/bin" ${CURRENT_PACKAGES_DIR}/tools/popsift/debug)
-    vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/popsift)
+     set(POPSIFT_TOOLS popsift-demo)
+     vcpkg_copy_tools(TOOL_NAMES ${POPSIFT_TOOLS} AUTO_CLEAN)
  endif()
 
 file(INSTALL ${SOURCE_PATH}/COPYING.md DESTINATION ${CURRENT_PACKAGES_DIR}/share/popsift RENAME copyright)
```
Comment (NancyLi1013) [inline ports/popsift/portfile.cmake:39]:
```suggestion
file(INSTALL ${SOURCE_PATH}/COPYING.md DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/popsift/portfile.cmake (hunk 4 above)
Comment (simogasp) [reply ports/popsift/portfile.cmake]: c3e8ee2bb

File: ports/popsift/portfile.cmake (hunk 3 above)
Comment (simogasp) [reply ports/popsift/portfile.cmake:18]: dc5557277

File: ports/popsift/portfile.cmake (hunk 2 above)
Comment (simogasp) [reply ports/popsift/portfile.cmake]: dc5557277

File: ports/popsift/portfile.cmake (hunk 1 above)
Comment (simogasp) [reply ports/popsift/portfile.cmake]: 7b68c4118

**Comment by simogasp** [comment]
Could you re-launch the CI? I don't think the failure is coming from this PR

**Comment by simogasp** [comment]
> /azp run

Thanks, but I think this time is another different pkg that is failing  :-(

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM now, thanks for your fixing @simogasp.

**Pull request merged**